tchannel/testing/vcr/__init__.py

```diff
@@ -20,10 +47,40 @@ def test_foo():
 from .cassette import Cassette
 
 
-def use_cassette(path):
+def use_cassette(path, record_mode=None):
+    """Use or create a cassette to record/replay TChannel requests.
+
+    This may be used as a context manager or a decorator.
+
+    .. code-block:: python
+
+        from tchannel.testing import vcr
+
+        @vcr.use_cassette('tests/data/foo.yaml')
```

Comment:
I think VCR added the ability to auto-name the cassette... not sure if that's worth it or not.